content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -88,8 +88,7 @@ get the best experience from Kubeflow.
     ibmc-block-gold (default)   ibm.io/ibmc-block   65s
     ```
 
-    Make sure `ibmc-block-gold` is the only `default` storageclass. If there are two or more rows in the above output, there is other `default` storageclass. Unset it with the below command, for example, to remove the previous `default` storageclass for the cluster.
-
+    Make sure `ibmc-block-gold` is the only `(default)` storage class. If there are two or more rows in the above output, unset the previous `(default)` storage classes with the below command:
```

Comment:
Is there a reason why you have round brackets now around the word `default`?

Reply:
`with the below command:` -> `with the command below:`

Reply:
`ibmc-block-gold (default)` is how kubectl indicate the default storage class. However, some Kubernetes cluster also comes with a storage class called `default` (without parentheses) but is not the default storage class. Therefore I'm adding the parentheses here to avoid confusion.

Reply:
☝️ is exactly what happens from my k8s cluster in IKS